automobile/buildings.md

```diff
@@ -75,9 +82,11 @@ normally found in a city environment:
 - BusStop
 - Fence
 - Fountain
+- PublicToilet
 
-%figure "The BusStop PROTO"
+%figure "The BusStop and PublicToilet PROTOs"
 
 ![bus_stop.png](images/bus_stop.png)
+![public-toilet.png](images/public-toilet.png)
```

Comment:
I would slightly reduce the size of the public toilet (the bus stop looks small compared to the toilets).